app/Providers/AuthServiceProvider.php

```diff
@@ -24,7 +22,8 @@ class AuthServiceProvider extends ServiceProvider
     public function boot()
     {
         $this->registerPolicies();
-
-        //
+        Auth::provider('admin-user-provider', function ($app, array $config) {
+            return new AdminUserProvider($app['hash'], $config['model']); 
```

Comment:
Where is it? Like, where's the actual class AdminUserProvider?

Reply:
app/Libraries

Reply:
The file is not commited.

Reply:
https://github.com/DevvitIO/CMS/tree/master/app